### ISSUE TITLE
[BUG] `crc setup` fails on non-english Windows

### ISSUE THREAD

**Issue by kadel** [code-ready/crc#3084]
### General information

  * OS:  Windows
  * Hypervisor:  Hyper-V 
  * Did you run `crc setup` before starting it (Yes/No)? yes
  * Running CRC on: Desktop pc

## CRC version
```bash
CodeReady Containers version: 2.0.1+f0fcd50e
OpenShift version: 4.10.3
Podman version: 3.4.4
```
  
## CRC status
```bash
DEBU CodeReady Containers version: 2.0.1+f0fcd50e
DEBU OpenShift version: 4.10.3
DEBU Podman version: 3.4.4
DEBU Running 'crc status'
DEBU Checking file: C:\Users\user\.crc\machines\crc\.crc-exist
Machine does not exist. Use 'crc start' to create it
```

## CRC config
```bash
- consent-telemetry                     : no
```

## Host Operating System
```bash
Host Name:                 DESKTOP-ODLC6DG
OS Name:                   Microsoft Windows 11 Pro
OS Version:                10.0.22000 N/A Build 22000
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          user
Registered Organization:
Product ID:                00330-80000-00000-AA532
Original Install Date:     25.03.2022, 14:30:58
System Boot Time:          25.03.2022, 15:00:13
System Manufacturer:       Microsoft Corporation
System Model:              Virtual Machine
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: AMD64 Family 23 Model 113 Stepping 0 AuthenticAMD ~3800 Mhz
BIOS Version:              Microsoft Corporation Hyper-V UEFI Release v4.1, 05.05.2021
Windows Directory:         C:\Windows
System Directory:          C:\Windows\system32
Boot Device:               \Device\HarddiskVolume1
System Locale:             cs;Čeština
Input Locale:              cs;Čeština
Time Zone:                 (UTC+01:00) Praha, Bratislava, Budapešť, Bělehrad, Lublaň
Total Physical Memory:     15 999 MB
Available Physical Memory: 13 649 MB
Virtual Memory: Max Size:  18 943 MB
Virtual Memory: Available: 16 456 MB
Virtual Memory: In Use:    2 487 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\DESKTOP-ODLC6DG
Hotfix(s):                 4 Hotfix(s) Installed.
                           [01]: KB5004342
                           [02]: KB5010690
                           [03]: KB5007215
                           [04]: KB5006755
Network Card(s):           1 NIC(s) Installed.
                           [01]: Microsoft Hyper-V Network Adapter
                                 Connection Name: Ethernet
                                 DHCP Enabled:    Yes
                                 DHCP Server:     172.25.64.1
                                 IP address(es)
                                 [01]: 172.25.66.123
                                 [02]: fe80::84df:970d:52af:f686
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.
```

### Steps to reproduce

  1. install crc
  2. run `crc setup`

### Expected
successful  crc setup

### Actual
```
PS C:\Users\user> crc setup
INFO Using bundle path C:\Users\user\.crc\cache\crc_hyperv_4.10.3_amd64.crcbundle
INFO Checking if current user is in Hyper-V Admins group
INFO Adding current user to Hyper-V Admins group
INFO Will run as admin: adding current user to Hyper-V administrator group
Please reboot your system and run 'crc setup' to complete the setup process
```
after reboot and running the command again, I get the same message

```
PS C:\Users\user> crc setup
INFO Using bundle path C:\Users\user\.crc\cache\crc_hyperv_4.10.3_amd64.crcbundle
INFO Checking if current user is in Hyper-V Admins group
INFO Adding current user to Hyper-V Admins group
INFO Will run as admin: adding current user to Hyper-V administrator group
Please reboot your system and run 'crc setup' to complete the setup process
```

This was on Windows 11 that was installed using Czech language.
If I do a fresh installation using English version ISO everything works ok.

I suspect that this might have something to do with Windows translating even names of groups.

![image](https://user-images.githubusercontent.com/57206/160138758-cfe5f57b-4e68-4acf-a91f-3142327a383c.png)

When I added my user to "Správci technologies Hyper-V" myself, the `crc setup` worked and I was able to use crc.

### Logs

```
PS C:\Users\user> crc setup -o debug
INFO Using bundle path C:\Users\user\.crc\cache\crc_hyperv_4.10.3_amd64.crcbundle
INFO Checking if current user is in Hyper-V Admins group
INFO Adding current user to Hyper-V Admins group
INFO Will run as admin: adding current user to Hyper-V administrator group
invalid format: debug
PS C:\Users\user> crc setup --log-level debug
DEBU CodeReady Containers version: 2.0.1+f0fcd50e
DEBU OpenShift version: 4.10.3
DEBU Podman version: 3.4.4
DEBU Running 'crc setup'
INFO Using bundle path C:\Users\user\.crc\cache\crc_hyperv_4.10.3_amd64.crcbundle
DEBU Error changing C:\Users\user\.crc\cache permissions to 0775
INFO Checking if current user is in Hyper-V Admins group
DEBU Running '$sid = New-Object System.Security.Principal.SecurityIdentifier("S-1-5-32-578")
        @([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole($sid)'
DEBU User is not a member of the Hyper-V administrators group
INFO Adding current user to Hyper-V Admins group
DEBU Running '(New-Object System.Security.Principal.SecurityIdentifier("S-1-5-32-578")).Translate([System.Security.Principal.NTAccount]).Value'
INFO Will run as admin: adding current user to Hyper-V administrator group
DEBU Running 'C:\Users\user\AppData\Local\Temp\crcScripts552661283\runAsAdmin.ps1'
Please reboot your system and run 'crc setup' to complete the setup process
```

**Comment by cfergeau** [comment]
This used to be an issue a long time ago which was fixed with https://github.com/code-ready/machine/commit/f748ac13
This code still seems to be present https://github.com/code-ready/crc/blob/aa7e94fb42664ab32afb007ab51597d571a3f984/pkg/crc/preflight/preflight_checks_windows.go#L101-L135 but maybe it has a bug.

**Comment by gbraad** [comment]
we specifically use the `sid` to look up the group:

```
'$sid = New-Object System.Security.Principal.SecurityIdentifier("S-1-5-32-578")
```

this got fixed and was confirmed on czech and german. perhaps a regression due to Windows 11 and a different powershell or group sid?

**Comment by s-gros** [comment]
I encountered the same issue on Windows 10 Pro 10.0.19042 using the German system locale. Manually adding the user to the group did resolve the issue.

I did some digging and it looks like ` fixUserPartOfHyperVAdmins` fails to retrieve the proper group name on this [line](https://github.com/code-ready/crc/blob/main/pkg/crc/preflight/preflight_checks_windows.go#L127). 
The group name obtained from `(New-Object System.Security.Principal.SecurityIdentifier("S-1-5-32-578")).Translate([System.Security.Principal.NTAccount]).Value` is  `VORDEFINIERT\Hyper-V-Administratoren`. The `BUILTIN\` prefix seems to be translated as well.

A fix might be to pass the `-SID` argument to `Add-LocalGroupMember` instead of the group name.

```
CodeReady Containers version: 2.0.1+f0fcd50e
OpenShift version: 4.10.3
Podman version: 3.4.4
```
